obsei/preprocessor/text_cleaner.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from typing import Any, Dict, List, Optional
+import string
+import re
+from unicodedata import normalize as unicode_decode
+
+import nltk
+from nltk.corpus import stopwords
+from nltk.stem import PorterStemmer
+from nltk.tokenize import word_tokenize
+
+from pydantic import PrivateAttr
+
+from obsei.analyzer.base_analyzer import AnalyzerRequest
+from obsei.preprocessor.base_text_cleaner import (
+    BaseTextCleaner,
+    BaseTextCleanerConfig,
+)
+
+logger = logging.getLogger(__name__)
+nltk.download('punkt')
+
+
+class TextCleaner(BaseTextCleaner):
+
+    def __init__(self, **data: Any):
+        super().__init__(**data)
+        self.stop_words: List = stopwords.words('english')
```

Comment:
Language should be user supplied via parameter. Otherwise default value should be english

Reply:
done